roles/config-python/tasks/main.yml

```diff
@@ -0,0 +1,17 @@
+# We can install specify version gcc/g++ in task, like this:
```

Comment:
it is better to check if python3 has already installed and skip 'Install python' task if yes.